bolt/lib/Rewrite/DWARFRewriter.cpp

```diff
@@ -767,17 +769,28 @@ void DWARFRewriter::updateDebugInfo() {
     CUPartitionVector PartVec = partitionCUs(*BC.DwCtx);
     for (std::vector<DWARFUnit *> &Vec : PartVec) {
       DIEBlder.buildCompileUnits(Vec);
-      for (DWARFUnit *CU : DIEBlder.getProcessedCUs())
-        processUnitDIE(CU, &DIEBlder);
+      llvm::SmallVector<DWARFUnit *> CompileUnits(
```

Comment:
Why is this necessary?

Reply:
Since we have two loops we only call DIEBlder.getProcessedCUs() once instead of in both loops.